src/main/options-normalizer.js

```diff
@@ -118,6 +118,9 @@ function optionInfoToSchema(optionInfo, { isCLI, optionInfos }) {
         parameters.preprocess = value => Number(value);
       }
       break;
+    case "string":
```

Comment:
I think it'd be better to have a way to specify a user-friendly type name in the help message, it should display `--server <url>` instead of `--server <string>` for example, but this is not a requirement so it could be done in the future.

Reply:
I agree that it'd be nicer UX for the user. However, right now, `int` type shows `<int>` in the CLI so this change would be in line with that. A cursory glance at `vnopts` doesn't show me an obvious way to change that option either, so I'd prefer to keep this PR small, and change the type name in the future.